addons/website_livechat/static/tests/thread_patch.test.js

```diff
@@ -31,7 +31,7 @@ test("Rendering of visitor banner", async () => {
     pyEnv["website.visitor"].write([visitorId], {
         display_name: `Visitor #${visitorId}`,
     });
-    const guestId = pyEnv["mail.guest"].create({ name: "Visitor 11" });
+    const guestId = pyEnv["mail.guest"].create({ name: `Visitor #${visitorId}` });
```

Comment:
why changed this line?

Reply:
The visitor id is `1` so forging `11` showed "Visitor 1" and "Visitor 11" in 2 parts of UI. The auto-compute of guest name is lacking code of mock server.